src/docs/references/transaction-process-actions/index.md

```diff
@@ -25,7 +25,9 @@ reference article.
 
 ## Actions
 
-### :action.initializer/init-listing-tx
+### Built-in
```

Comment:
How about "Transaction initialization"? That communicates better the purpose of the action(s) here.